clappr/clappr-android
# fix(live_error): Changing error catching

### Goal
To solve the problem reported: When the user is in a bad internet connection for few seconds the player lose the transmission window for live content. Today when it happens the player returns a generic error.
This code makes the player recover itself in this case.

### How to test
1 - Execute any LIVE video
2 - Make your internet connection very bad for few seconds
3 - See in the log that the `BehindLiveWindowException` happens without visual impact.
